elixir/apps/web/lib/web/live/sites/new_token.ex

```diff
@@ -78,11 +74,12 @@ defmodule Web.Sites.NewToken do
           <.tabs :if={@env} id="deployment-instructions">
             <:tab
               id="systemd-instructions"
+              icon="hero-command-line"
               label="systemd"
               phx_click="tab_selected"
               selected={@selected_tab == "systemd-instructions"}
             >
-              <p class="p-4">
+              <p class="p-6">
```

Comment:
This aligns the content of the tabs component with the content inside the code blocks